x-pack/plugins/cases/server/client/cases/update.ts

```diff
@@ -97,6 +97,22 @@ function throwIfUpdateTypeCollectionToIndividual(
   }
 }
 
+/**
+ * Throws an error if any of the requests attempt to update the type of a case.
+ */
+function throwIfUpdateType(requests: ESCasePatchRequest[]) {
```

Comment:
Can we change the type of a case if `subCasesEnabled`? Is it a valid use case?

Reply:
Yeah we allow changing an individual to collection but not collection to individual.